packages/api-fetch/src/index.js

```diff
@@ -156,6 +157,42 @@ function apiFetch( options ) {
 	} );
 }
 
+/**
+ * Function that fetches data using apiFetch, and updates the status.
+ *
+ * @param {string} path Query path.
+ */
+function useApiFetch( path ) {
```

Comment:
By naming standards, this should be `useAPIFetch`.

See: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#abbreviations-and-acronyms